### PR TITLE
Update readme: explain shopper tokens

### PR DESCRIPTION
**Pull request by tjukes**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -188,6 +188,22 @@ The response will be similar to the following. Note that this is just a simulati
 )
 ```
 
+### Shopper Tokens
+
+If you are using our [Drop In Gift Card](https://www.lightrail.com/docs/#drop-in-gift-cards/quickstart) solution, you can use this library to generate shopper tokens for transacting against a customer's account.
```
Comment (Tim-Jordison) [inline README.md]: Drop In Gift Card -> Drop-in Gift Card

File: README.md [hunk 2]
```diff
@@ -188,6 +188,22 @@ The response will be similar to the following. Note that this is just a simulati
 )
 ```
 
+### Shopper Tokens
+
+If you are using our [Drop In Gift Card](https://www.lightrail.com/docs/#drop-in-gift-cards/quickstart) solution, you can use this library to generate shopper tokens for transacting against a customer's account. 
+
+A shopper token is generated from a a unique customer identifier from your system: this is the same `shopperId` or contact `userSuppliedId` you would have used when creating the Contact. If you haven't yet created a Contact record, the function will create one for you automatically based on this ID. (Note that if you want extra information to be associated with the Contact, like their name or email address, you should [create the contact](#handling-contacts) first.)
```
Comment (Tim-Jordison) [inline README.md]: Tiny bit confused by "this is the same `shopperId` or ...". Would someone have seen the term `shopperId` before? Just curious if this should be more along the lines of "A shopper token is generated from a unique customer identifier from your system which we'll refer to as the shopperId. When using a shopperToken it'll create a Contact record using the `shopperId` as the Contact object's `userSuppliedId`...

There's also an extra a in the sentence. "from a a unique".

Comment (tjukes) [reply README.md]: Shopper IDs are mentioned in the contact creation section near the top, and again in the account creation section. Things should be clear but not overly duplicated I think. Depends on which side of that line we think this falls on.

Comment (pushplay) [reply README.md]: A contact can be identified by one of three things: contactId, userSuppliedId or shopperId.  You're missing the first one.

Some actions using the shopper token might create a contact for you but the function that generates the shopper token doesn't.  That's an important distinction.

Comment (tjukes) [reply README.md]: Good catch, thanks.

**Pull request merged**